src/style/colors.js

```diff
@@ -123,11 +123,43 @@ class Colors {
       tints.push(tint);
     });
 
-    const sliced = tints.slice(0, 8);
-    return sliced;
+    let sliced = tints.slice(0, 8);
+    const adjusted = adjustSaturation(sliced, color);
+    return adjusted || sliced;
   });
 }
 
+function adjustSaturation(colors, color) {
+  let array;
+  const lightnessLevel = 80;
+  const saturationLevel = 60;
+  const hsl = Color(color).hsl();
+  const lightness = Math.round(hsl.color[2]);
+
+  if (lightness > lightnessLevel) {
+    const saturation = Math.round(hsl.color[1]);
+    if (saturation > saturationLevel) {
+      array = [];
```

Comment:
sorry, should have wrote it before
let's shorten this a little..
you can
`array = _.map(colors, e => e !== color ? addSaturation(e, saturationLevel) : e)`

Reply:
Cool